content/id/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Mengelola Resource
```

Comment:
```suggestion
title: Mengelola Sumber Daya
```

Reply:
Should we use _resource_ to refer k8s resource instead?

Reply:
resource is not Kubernetes specific, so I think we can translate this word according to Mas Ivan Lanin's [glosarium](https://github.com/jk8s/sig-docs-id/blob/master/resources/RSNI-glossarium.pdf)

Reply:
however renaming resource to "sumber daya" is nonblocking merge since it might require lots of effort to replace everything in this page